mailpoet/mailpoet
# Feat/design the landing page - MAILPOET-4799

## Description
This PR adds the final content for the landing page.

![mailpoet-landingpage](https://user-images.githubusercontent.com/30554163/212331065-5c247819-df52-436c-9bfe-e1929fa0453a.png)


## QA notes

You can access the page here: `wp-admin/admin.php?page=mailpoet-landingpage`

You need to enable Landingpage experimental features here: `/wp-admin/admin.php?page=mailpoet-experimental`

Check https://github.com/mailpoet/mailpoet/pull/4620 for more QA information. 

When testing using the ZIP build, the images on the page won't load because they will be publicly available after the release.

You can check the feature with the images by testing with the dev env and adding `define( 'MAILPOET_USE_CDN', false ); ` to the wp-config.php. Alternatively, you can ping me, and I'll push the images to the repo so that they also work from the zip.



## Linked tickets

[MAILPOET-4799](https://mailpoet.atlassian.net/browse/MAILPOET-4799)




[MAILPOET-4799]: https://mailpoet.atlassian.net/browse/MAILPOET-4799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
